### DISCUSSION TITLE
Remove conversation module as a requirement or offer a version without it

### DISCUSSION THREAD

**Discussion by MoHf7f4** [zachowj/hass-node-red#293] [Feature Requests]
Is the conversation module really needed for this to work?  

It forces module to load if this is installed.  That is added to the UI of HA and can't be removed.

**Comment by zachowj** [comment]
The conversation module is essential for the sentence node to operate.

I don't intend to release two separate versions. However, you can remove the conversation dependency from the after_dependencies section in the manifest.json file. As long as you don't use the sentence node, this change shouldn't cause any errors.

<https://github.com/zachowj/hass-node-red/blob/df2c9c4a4d6ebee4b8a231020c51214e3373f9eb/custom_components/nodered/manifest.json#L5>
